### DISCUSSION TITLE
2009/08/04/%E5%BC%AF%E6%9B%B2%E7%9A%84%E6%95%B0%E6%8D%AE%E8%BF%9E%E7%BA%BF

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#271] [Announcements]
# 2009/08/04/%E5%BC%AF%E6%9B%B2%E7%9A%84%E6%95%B0%E6%8D%AE%E8%BF%9E%E7%BA%BF

本文内容出自LVAV论坛的一片帖子：http://lavag.org/topic/10670-wired-wires/page\\p\\63544&#entry63544。

https://blog.qizhen.xyz/2009/08/04/%E5%BC%AF%E6%9B%B2%E7%9A%84%E6%95%B0%E6%8D%AE%E8%BF%9E%E7%BA%BF

**Comment by ruanqizhen** [comment]
BO
2009年08月5日 00:10 编辑
这幅图可以用来给vi script做广告了。

回复
Qizhen
2009年08月4日 15:52 编辑
没什么实用，只是看着爽。

回复
Jay
2009年08月4日 15:10 编辑
连线弄得这么绕，有什么特殊用处吗？弯折的直线不是挺好的嘛

回复
